test/test_onnx.py

```diff
@@ -79,7 +79,8 @@ def to_numpy(tensor):
         inputs = list(map(to_numpy, inputs))
         outputs = list(map(to_numpy, outputs))
 
-        ort_session = onnxruntime.InferenceSession(onnx_io.getvalue())
+        ort_session = onnxruntime.InferenceSession(onnx_io.getvalue(),
+                                                   providers=onnxruntime.get_available_providers())
```

Comment:
Lint is failing: https://github.com/pytorch/vision/actions/runs/6271822144/job/17032160789?pr=7982#step:11:410

```suggestion
        ort_session = onnxruntime.InferenceSession(onnx_io.getvalue(), providers=onnxruntime.get_available_providers())
```